docs/api/core/decorator.md

```diff
@@ -419,7 +419,7 @@ name if JS engine doesn't implements ES6 name property
 
 An alternative and more declarative way to using the `inputs` property on `@Component`/`@Directive`.
 
-Binds to Component/Directive via on of binding types `=`/`<`/`@`.
+Binds to Component/Directive via on of binding types `=`/`<`/`@` with the optional `?` alias.
```

Comment:
I would rather use `with the optional `?` alias in Angular 1 syntax terms`. Because as I've mentioned I'm binding things myself